### PR TITLE
Feature/spleeter2update

### LINKED ISSUE #53: Trivial update to spleeter2?
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Pull request provided with udpates to the js for the spleeter 2 invocation (arguments changed)
 
**Describe alternatives you've considered**
No alternatives needed, seems to work out of the box, unless theres a reason to hold back the version? Simplifies the install steps as dependencies are resolved automatically.

**Additional context**
None

### PR DESCRIPTION
**Pull request by bytebraid**
resolves #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,7 @@ if you want to verify everything is installed on windows, download https://raw.g
 brew install python@3.7
 brew link --force python@3.7
 brew install ffmpeg
-pip3 install spleeter==1.5.3
-pip3 install numba==0.48.0
-pip3 install numpy==1.18.5
+pip3 install spleeter
```
Comment (diracdeltas) [inline README.md]: ++ thanks for confirming this works. could you please pin a version number in this line? (to make sure that breaking updates in the future don't break things)

Comment (diracdeltas) [reply README.md]: looks like 2.1.0 is the version we want

**Comment by bytebraid** [comment]
Pushed the change but shortened it the minor revision (2.1) as by
definition there shouldn't be any breaking changes in a patch release...!

Cheers,
Ciarán

**Comment by bytebraid** [comment]
Despite what I said about patch releases in the previous email I'm not
100% on pip behaviour for patch versions... presumably they're defined
by the package owner. Nevertheless running:

`pip3 install spleeter`

currently installs 2.1.2 on my system

however

`pip3 install spleeter==2.1`

favourably installs 2.1.0 and ignores the latest available patch

Not sure what's going on with that but hopefully it doesn't matter a
huge deal for this change... let me know otherwise

Cheers,
Ciarán

**Review by diracdeltas — APPROVED** [review]
thanks! fingers crossed that the patch version won't matter.

**Comment by diracdeltas** [comment]
@skircher could you please download the zip from https://github.com/diracdeltas/spleeter4max/releases/tag/1.5 and let me know if it works? thank you!

**Pull request merged**